src/cdk/construct-lib.ts

```diff
@@ -7,11 +7,11 @@ export interface ConstructLibraryOptions extends JsiiProjectOptions {
    *
    *     https://awscdk.io/packages/[@SCOPE/]PACKAGE@VERSION
    *
-   * The catalog will also post a tweet to https://twitter.com/awscdkio with the
+   * The catalog will also post a tweet to https://x.com/awscdkio with the
    * package name, description and the above link. You can disable these tweets
    * through `{ announce: false }`.
    *
-   * You can also add a Twitter handle through `{ twitter: 'xx' }` which will be
+   * You can also add a X handle through `{ X: 'xx' }` which will be
```

Comment:
```suggestion
   * You can also add a X handle through `{ twitter: 'xx' }` which will be
```
This is an external API the can't changed unilaterally from here.